index.html

```diff
@@ -0,0 +1,219 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Timothy | Portfolio</title>
+</head>
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <a href="#home">
+        <div class="logo">Timothy</div>
+      </a>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```

Comment:
- Kindly , The hamburger button is has a `nav` tag for semantics reason. as your previous code reviwer seggested you 👌👌😍